Makefile

```diff
@@ -102,6 +103,13 @@ build-sdist:
 remove-generic-wheel:
 	rm -f dist/*-py3-*.whl
 
+# Package and publish the docs to causaLens' artifactory
+package-and-publish-docs:
+	poetry source add causalens https://causalens.jfrog.io/artifactory/api/pypi/python-open-source/simple/
+	poetry config http-basic.causalens $${ARTIFACTORY_USERNAME} $${ARTIFACTORY_PASSWORD}
```

Comment:
@andrew-causalens I think the $$ is causing your issue with the build. Can you try changing it to a single one instead, e.g. the publish step below taking pypi token?

Reply:
Here is the same block in `dara` btw: https://github.com/causalens/dara/blob/master/Makefile#L79

Reply:
Okay, I tried setting the creds again. Give it another try with the double $$ and see if it was a typo there.

Reply:
Thanks @sam-causalens! 🙏 . Kicked it off and will try single $ if it fails.

Reply:
Just to confirm it still [fails](https://github.com/causalens/cai-causal-graph/actions/runs/5925524641/job/16065113134) with $$:
```
poetry add --source=causalens docs-builder@~0.2.0
Source (causalens): Authorization error accessing https://causalens.jfrog.io/artifactory/api/pypi/python-open-source/simple/docs-builder/

Could not find a matching version of package docs-builder
```

Will try $ now.

Reply:
@sam-causalens it [fails](https://github.com/causalens/cai-causal-graph/actions/runs/5925564928/job/16065233038) with single $ too:
```
poetry add --source=causalens docs-builder@~0.2.0
Source (causalens): Authorization error accessing https://causalens.jfrog.io/artifactory/api/pypi/python-open-source/simple/docs-builder/

Could not find a matching version of package docs-builder
```

Can you check on JFrog to see if the requests are even making it to it and potentially get some further info from the logs?

Reply:
I tried recreating the secrets in github altogether. Can you switch back to $$ and try again?

Reply:
Still [fails](https://github.com/causalens/cai-causal-graph/actions/runs/5926898619/job/16069224948) with new secrets and back to $$.

Reply:
All good now. Thanks!